schemas/test/en/test_routing_and_skipping_section_dependencies_new_calculated_summary.json

```diff
@@ -79,6 +79,62 @@
                                 ]
```

Comment:
I think the existing guidance is wrong, it says:
```
If the total is equal to £100 a new section will appear on the hub and if it is greater than or equal to £100 a dependent question will appear in the dependent question section
```

According to the schema it should be `10` not 100 and `if it is less than 10 then it will appear`?

Reply:
Updated this 👍